src/System.Numerics.Vectors/src/System.Numerics.Vectors.csproj

```diff
@@ -0,0 +1,92 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp2.0;netstandard2.0;$(NetFrameworkMinimum)</TargetFrameworks>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <StrongNameKeyId>Microsoft</StrongNameKeyId>
+    <IsPackable>false</IsPackable>
+    <VersionPrefix>4.5.0</VersionPrefix>
+    <VersionPrefix Condition="'$(IsPackable)' == 'true'">4.6.0</VersionPrefix>
+    <AssemblyVersion>4.1.4.0</AssemblyVersion>
+    <!-- NO-INCREMENT: This version is frozen for .NET Standard and .NETCoreApp because the assembly ships inbox. -->
```

Comment:
```suggestion
    <AssemblyVersion>4.1.4.0</AssemblyVersion> <!-- NO-INCREMENT: This version is frozen for .NET Standard and .NETCoreApp because the assembly ships inbox. -->
```
We put this on the same line elsewhere: https://github.com/dotnet/maintenance-packages/blob/df0d9ffb05a586fdd6362c9ad5aff3135be9a64f/src/System.Memory/src/System.Memory.csproj#L11

Putting it on a new line makes it look like it applies to the following element which is not what we want.

Reply:
Yes, oops. My bad.